static/js/application-version.js

```diff
@@ -0,0 +1,46 @@
+
+
+$(document).ready(function () {
+    if (Cookies.get('theme')) {
+        theme = Cookies.get('theme');
+        $('body').attr('data-theme', theme);
+        
+    }
+    $('.theme-btn').on('click', themePickerHandler);
+    displayNavbar();
+    console.log("before testing")
+    fetchVersionInfo();
+    console.log("after testing")
```

Comment:
Remove these `console.log` lines

Reply:
I have made the changes as per requirements. Please review it